cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java

```diff
@@ -213,22 +213,38 @@ public Cap1xxx(Context context, String i2cName, String alertName, Configuration
         this(i2cName, alertName, chip, handler);
     }
 
+    /**
+     * Create a new Cap1xxx controller with the default I2C address.
+     *
+     * @param i2cName I2C port name where the controller is attached. Cannot be null.
+     * @param alertName optional GPIO pin name connected to the controller's
+     *                  alert interrupt signal. Can be null.
+     * @param chip identifier for the connected controller device chip.
+     * @param handler optional {@link Handler} for software polling and callback events.
+     * @throws IOException
+     */
+    public Cap1xxx(String i2cName, String alertName, Configuration chip, Handler handler) throws IOException {
+        this(i2cName, I2C_ADDRESS, alertName, chip, handler);
```

Comment:
This is a good change, but if we expose the I2C address as a parameter let's rename the constant to `DEFAULT_I2C_ADDRESS` to match our other drivers ([example](https://github.com/androidthings/contrib-drivers/blob/master/bmx280/src/main/java/com/google/android/things/contrib/driver/bmx280/Bmx280.java#L48))